### ISSUE TITLE
playlist stop my bot

### ISSUE THREAD

**Issue by LEGENDmathlava** [serenity-rs/songbird#92]
**Songbird version:** (version)
songbird = {version = "0.2.0", features = ["builtin-queue"]}

**Rust version (`rustc -V`):** (version)
rustc 1.57.0-nightly (497ee321a 2021-09-09)

**Serenity/Twilight version:** (version)
serenity = {version = "0.10.7", features = ["client", "framework", "standard_framework", "voice", "rustls_backend"]}

**Output of `ffmpeg -version`, `youtube-dl --version` (if relevant):**
ffmpeg: ffmpeg version 3.4.8-0ubuntu0.2
youtube-dl: 2021.06.06

**Description:**
Excuse me for google translate from Japanese,
And I'm also a beginner on github.

When I add a playlist song using `input :: ytdl (& url)` and the first song finishes, all songbird commands stop. Is this a bug?

Also, is there a way to put all the songs in the playlist into the queue at once at this time?

**Comment by FelixMcFelix** [comment]
I tried to reproduce something like this last night before implementing #96, but couldn't cause any odd behaviour. Can you explain exactly what commands stop when a track ends?

**Comment by 2vg** [comment]
> Also, is there a way to put all the songs in the playlist into the queue at once at this time?

no, to pass `--no-playlist` to the songbird's ytdl function when calling `youtube-dl`.
basically, it is recommended to create your own queue system instead of songbird's builtin queue.
